libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/OAuth/CSFSalesforceOAuthRefresh.m

```diff
@@ -30,6 +30,7 @@
 #import "SFOAuthCoordinator.h"
 #import "SFAuthenticationManager.h"
 #import "SFSDKEventBuilderHelper.h"
+#import "CSFNetwork.h"
```

Comment:
This change has no accompanying code to utilize it.  Can we remove the import statement?